src/features/CourtCaseFilters/CourtCaseFilter.tsx

```diff
@@ -47,6 +46,10 @@ const reducer = (state: Filter, action: FilterAction): Filter => {
       newState.lockedFilter.value = action.value
       newState.lockedFilter.label = action.value ? "Locked" : "Unlocked"
       newState.lockedFilter.state = "Selected"
+    } else if (action.type === "myCases") {
+      newState.myCasesFilter.value = action.value
+      newState.myCasesFilter.label = action.value ? "Locked to me" : undefined
```

Comment:
Just for future reference, alternatively you could just do
```suggestion
      newState.myCasesFilter.label = action.value && "Locked to me"
```